modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify the status of a boot source
+
+You can verify that a boot source is system-defined or user-defined.
+
+The `status` section of each boot source listed in the `status.dataImportChronTemplates` field of the `HyperConverged` CR indicates the type of boot source. For example, `commonTemplate: true` indicates a system-defined (`commonTemplate`) boot source. `status: {}` indicates a user-defined boot source.
+
+.Example of `status.dataImportChronTemplates` field of the `HyperConverged` CR
+
+[source,yaml]
+----
+...
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+...
+spec:
+  ...
+status: <1>
+  ...
+  dataImportCronTemplates: <2>
+  - metadata:
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+      name: centos-7-image-cron
+    spec:
+      garbageCollect: Outdated
+      managedDataSource: centos7
+      schedule: 55 8/12 * * *
+      template:
+        metadata: {}
+        spec:
+          source:
+            registry:
+              url: docker://quay.io/containerdisks/centos:7-2009
+          storage:
+            resources:
+              requests:
+                storage: 30Gi
+        status: {}
+    status:
+      commonTemplate: true <3>
+    ...
+  - metadata:
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+      name: user-defined-dic
+    spec:
+      garbageCollect: Outdated
+      managedDataSource: user-defined-centos-stream8
+      schedule: 55 8/12 * * *
+      template:
+        metadata: {}
+        spec:
+          source:
+            registry:
+              pullMethod: node
+              url: docker://quay.io/containerdisks/centos-stream:8
+          storage:
+            resources:
+              requests:
+                storage: 30Gi
+        status: {}
+    status: {} <4>
+...
+----
+<1> The `Status` field for the `HyperConverged` CR.
```

Comment:
The way you are showing the YAML now, the `status` field is lowercase.

Reply:
Corrected